orchestra/cmds/inspect.py

```diff
@@ -84,9 +92,44 @@ def handle_hash_material(args):
     return 0
 
 
+def handle_dependencies(args):
+    config = Configuration(use_config_cache=args.config_cache)
+    main_build = config.get_build(args.component)
+
+    if main_build is None:
+        suggested_component_name = config.get_suggested_component_name(args.component)
+        logger.error(f"Component {args.component} not found! Did you mean {suggested_component_name}?")
+        return 1
+
+    if args.runtime:
+        dependencies = [*main_build.dependencies]
+    else:
+        dependencies = [*main_build.dependencies, *main_build.build_dependencies]
+
+    builds_to_scan = [_get_installed_build(config, d) for d in dependencies]
```

Comment:
* Mmmmh, why the installed one?
  The other two commands in `orc inspect component` do not care about what's installed, why should this one care?
* I suggest to use a `set` for visited stuff, I think under certain conditions we might have dependency loops.

Reply:
1. You cannot get a proper list without assuming is the installed one, because dependencies are at the build level and not at the component level
2a. orchestra will promptly complain: https://github.com/revng/revng-orchestra/blob/master/orchestra/executor.py#L420
2b. Even if there was a loop the `if dependency not in dependencies` at line `117` would break away the second time a dependency is encountered

Reply:
What happens if it's not installed? It's not obvious why this shouldn't work if it's not installed. If it's not installed you could return the list you'd obtain if you where to `orc install $COMPONENT` or maybe a superset of all the actual dependencies (which is what `pip-package-list.py` basically does using `orc inspect component hash-material`).

If you want to keep it this way, maybe we should call this `orc inspect component installed-dependencies` or force the user to specify `--installed` and say that the version without `--installed` is not implemented yet.

Reply:
I'd go with the `--installed` approach, since it's what we want for our current goals.

Reply:
Change implemented